files/ids/bgee/stats.htm

```diff
@@ -272,13 +272,34 @@
 </div>
 <br />
 
-<div class="idsHeader"><a name="46">46 MORALEBREAK</a></div>
+<div class="idsHeader"><a name="MORALEBREAK">46 MORALEBREAK</a></div>
 <div class="indent1">
-Returns <code>0</code> or <code>1</code> if the creature is immune to fear.
+  <ul>
+    <li>This stat is the lower bound for avoiding a morale failure.</li>
+    <li>If <code><a href="../../../file_formats/ie_formats/cre_v1.htm#CREV1_0_Header_Morale">Morale</a></code> is lower than this value, morale failure occurs, if <code><a href="../../../file_formats/ie_formats/cre_v1.htm#CREV1_0_Header_Morale">Morale</a></code> is higher, any morale failure is restored.</li>
```

Comment:
Would be clearer to say it is *removed*, not restored.

Reply:
> Would be clearer to say it is _removed_, not restored.

OK, done.